comps/third_parties/nebula/deployment/kubernetes/community_edition.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: apps.nebula-graph.io/v1alpha1
+kind: NebulaCluster
+metadata:
+  name: nebula
+  #namespace: nebula-operator-system
+spec:
+  graphd:
+    resources:
+      requests:
+        cpu: "500m"
+        memory: "500Mi"
+      limits:
+        cpu: "1"
+        memory: "1Gi"
+    replicas: 1
+    image: vesoft/nebula-graphd
+    #version: v3.1.3
+    service:
+      type: NodePort
+      externalTrafficPolicy: Local
+    logVolumeClaim:
+      resources:
+        requests:
+          storage: 2Gi
+      storageClassName: "${STORAGE_CLASS_NAME}"
```

Comment:
If nebula requires k8s to have a valid storage class, we'd better to say it clearly in README. Also if we follow the README strictly, this `${STORAGE_CLASS_NAME}` dereference could cause the `kubectl create -f enterprise_edition.yaml` fail due to the mismatch of storageClassName. Does nebula operator support not setting specific storageClassName?